sim/druid/ferocious_bite.go

```diff
@@ -48,7 +50,8 @@ func (druid *Druid) registerFerociousBiteSpell() {
 		},
 
 		ApplyEffects: core.ApplyEffectFuncDirectDamage(core.SpellEffect{
-			ProcMask: core.ProcMaskMeleeMHSpecial,
+			ProcMask:        core.ProcMaskMeleeMHSpecial,
+			BonusCritRating: t9bonus,
 			DamageMultiplier: 1 +
 				0.03*float64(druid.Talents.FeralAggression) +
 				core.TernaryFloat64(druid.HasSetBonus(ItemSetThunderheartHarness, 4), 0.15, 0),
```

Comment:
Additive damage multipliers. Usually Wow uses multiplicative, so just making sure this is correct

Reply:
yea multiply is probly correct, will swap it